public/gallery.html

```diff
@@ -106,12 +124,28 @@
 			<button class="bg-black text-white text-lg ring-2 py-1 px-4 rounded-lg self-center" type="submit"><strong>Apply</strong></button>
 		</div>
 	</div>
-<!-- End filters container -->
 </div>
+<!-- End filters container -->
+
 <!-- Menu end -->
-<div>
-	<img  src="Assets/images/gallery/percare_gallery_hero.png" alt="Animals posing in a art gallery">
+
+<!-- Grid section -->
+
+<!-- Grid container start -->
+<div id="gridBackground">
+	<div class="w-full h-full grid grid-cols-3 grid-rows-3 gap-4" id="gridContainer">
+		<!-- Grid box container -->
+		<div class="imageHoverContainer relative">
+			<img src="Assets/images/gallery/gallery_grid/dog-1.png" alt="dog portrait">
+			<div class="statsBox flex flex-col justify-center items-center  absolute bottom-0 inset-x-0 bg-purple-400 h-24 bg-opacity-90 opacity-0">
+				<div class="text-15 font-bold text-blue-800">Gender: male</div>
+				<div class="text-15 font-bold text-blue-800">Rating: 4☆</div>
```

Comment:
While this here is info useful for the end-user, it will not help you much for filtering purposes

Reply:
right, that will not be part of the functionality... it is only for the user to see.
Im gonna use something like `data-gender` and `data-rating` to filter the items.
I also understand that the rating values should update automatically when user's vote but im not quite there yet :)